scripts/screeps_rl_training_runner.py

```
@@ -3191,13 +3698,14 @@ def policy_gradient_candidate_parameters_metadata_only(policy_gradient: JsonObje
-    if runtime_injection is False:
-        return True
-    inline_applied = first_present(
-        support,
-        ("inline_candidates_applied_to_simulator", "inlineCandidatesAppliedToSimulator"),
+    return (
+        runtime_injection is True
+        or scope == "runtime_injected"
+        or scope == "partial_runtime_injection"
+        or status == "partial"
+        or status == "not_injected"
+        or reward_use == "eligible_with_evaluated_runtime_parameters"
```

Ruff (0.15.13)

[warning] Consider merging multiple comparisons.

Merge multiple comparisons

(PLR1714)